src/index.jsx

```diff
@@ -14,10 +14,12 @@ import appMessages from './i18n';
 import './index.scss';
 import ProgramRecordsList from './components/ProgramRecordsList';
 import ProgramRecord from './components/ProgramRecord';
+import Head from './components/Head';
 
 subscribe(APP_READY, () => {
   ReactDOM.render(
     <AppProvider>
+      <Head />
```

Comment:
I'm sorry, but I'm confused by this. If the HelmetProvider is a context, then I would expect it to wrap everything that might modify the head.

<AppProvider>
  <HelmetProvider context={helmetContext}>
     <Head />
     <Header/>
     ... everything else...
  </HelmetProvider>
</AppProvider>

That way, anything that accesses the header is in the context of the provider.

And then the Head implementation contains a Helmet instance that wraps the header metadata.

Am I misunderstanding?

Reply:
In this case, other components can't modify the favicon and title tag, because the values come directly from the config (comes from `AppProvider`), that's why I  don't think the provider has to wrap all the components inside the app. 

Let me know if that makes sense either I'm okay to make the change.

Reply:
Ah. I see. I think that it would be best to still use the pattern where the app is wrapped for a couple of reasons, but I can see where they don't seem related to the problem you are solving:
1) This is the first use of react-helmet-async, and will end up being copy and pasted into other projects as an example. Other projects may use it for other reasons.
2) If the use of it is expanded in the future for header metadata that might be modified by the application, having the context wrapping it will make those changed "just work", eliminating bugs and debugging.

If you're ok to make the change, I would really appreciate it. Thanks.

Reply:
I made the change, could you take a look, please? @cdeery